packages/backend/src/house/util/house.util.ts

```diff
@@ -0,0 +1,11 @@
+// Generate random house code
+export function generateString(length) {
+  const characters =
+    'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+  let result = '';
+  const charactersLength = characters.length;
+  for (let i = 0; i < length; i++) {
+    result += characters.charAt(Math.floor(Math.random() * charactersLength));
+  }
+  return result;
+}
```

Comment:
By any chance was this function inspired from this Stackoverflow answer? https://stackoverflow.com/a/1349426
If so, it is probably best to attribute it to the author somewhere in the code.